src/main/resources/templates/kinbin.html

```diff
@@ -27,12 +26,17 @@
             display: inline-block;
             margin-left: 30px;
         }
+
+        .column {
+            display: inline-block;
+            padding-left: 30px;
+        }
     </style>
 </head>
 <body>
     <h1>Kinbin's Adventure</h1>
     <div id="buttons">
-        <button id="add-column">Add new column</button>
+        <button id="add-column" onclick="addColumn()">Add new column</button>
```

Comment:
why not a simple link?

Reply:
where?

Reply:
This button has the same behaviour of a link: opens another page.

Reply:
onclick="<link>" will work? If yes, or I forgot it or I didn't know xD

Reply:
For a while, I will keep it, because I'm not sure if links will change and I prefer keep them in single file (in this case, in JavaScript).

Reply:
Actually, I was suggesting something like
```
<a href="/add_column_form">Add new column</a>
```
because you are using javascript for something that is the function of an existent html element. About the links, spring doesn't have something that creates the urls of the pages by the name of the method in the Controller?

Reply:
Ah, that's true. The button acts as a link and I didn't spot it. When we can manage to make this code private, I can fix it.
About Spring Controller, can you explain me better the question?